src/preamble.js

```diff
@@ -433,7 +433,15 @@ function UTF8ArrayToString(u8Array, idx) {
   while (u8Array[endPtr]) ++endPtr;
 
   if (endPtr - idx > 16 && u8Array.subarray && UTF8Decoder) {
-    return UTF8Decoder.decode(u8Array.subarray(idx, endPtr));
+    var buf;
```

Comment:
(this line could be merged to below for `var buf = ...` in both cases to save a few chars, but not a biggie)